js/components/addressBook/addressBook_controller.js

```diff
@@ -3,6 +3,9 @@ angular.module('contactsApp')
 	var ctrl = this;
 
 	ctrl.showUrl = false;
+	/* globals oc_config */
+	/* eslint camelcase: [2, {properties: "never"}] */
+	ctrl.canExport = parseFloat(oc_config.version.substring(0, 3)) >= 9.1;
```

Comment:
float is not really the right thing to use - generally speaking since 9.1 and 9.10 are equal in terms of float but not in terms of version. Anyway - there will never be a version higher then 9.2.

BUT: 9.1 is not the correct value to test against. you need to test against 9.0.2 because the fix has been backported